Build issues: `npm test` fails with assertion error

I would like to fork this add-on in order to create a similar container add-on with a different set of domains. I've cloned it on my machine and I tested the `1.3.1` tag and the `master` branch with `npm ci` and then `npm test` and I got the following error:

```
  1) Already Open Reopen
       Add-on initializes with already open but still loading Tab
         should wait for still loading tabs and then reopen them:
     AssertionError: expected stub to have been called with arguments matching { cookieStoreId: "firefox-container-5", url: "https://www.facebook.com" }
      at Context.it (test/features/already-open-reopen.test.js:69:59)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

--- a/test/features/already-open-reopen.test.js
+++ b/test/features/already-open-reopen.test.js
@@ -60,11 +60,11 @@ describe("Already Open Reopen", () => {
     it("should wait for still loading tabs and then reopen them", async () => {
       expect(background.browser.tabs.create).to.not.have.been.called;
       tab.url = "https://www.facebook.com";
-      const [promise] = background.browser.tabs.onUpdated.addListener.yield(tab.id, {
+      background.browser.tabs.onUpdated.addListener.yield(tab.id, {
         url: tab.url,
         status: "complete"
       }, tab);
-      await promise;
+      await new Promise(setTimeout);
 
       expect(background.browser.tabs.create).to.have.been.calledWithMatch({
         url: "https://www.facebook.com",

